MsCorePkg/Universal/StatusCodeHandler/Serial/Common/SerialStatusCodeHandler.c

```diff
@@ -135,18 +137,15 @@ SerialStatusCode(
     //
     // EFI_STATUS_CODE_STRING_DATA
     //
-    CharCount = AsciiSPrint (
-                  Buffer,
-                  sizeof (Buffer),
-                  "%a\n\r",
-                  ((EFI_STATUS_CODE_STRING_DATA *) Data)->String.Ascii
-                  );
+    BufferPtr = ((EFI_STATUS_CODE_STRING_DATA *) Data)->String.Ascii;
```

Comment:
Will this "in-place" Ascii string have the newlines?
Are the newlines necessary?

Reply:
Yes, it wil, and no they are not. This is actually the main reason I'm in here; FSP uses this StatusCode Data Type to print things. And if you add a new line to each thing it print, then:

```
for (i=0; i<something; i++) {
  DEBUG((DEBUG_INFO, "0x%02x ", data[i]));
}
```

looks like this:

```
0xDE
0xAD
0XBE
0xEF
```

instead of

`0xDE 0xAD 0xBE 0xEF `

as expected.

Reply:
Honestly, if we need to minimize changes here for some reason, simply removing the \n\r would solve the main issue.